frb_codegen/src/generator/rust/ty_enum.rs

```diff
@@ -276,12 +277,32 @@ impl TypeRustGeneratorTrait for TypeEnumRefGenerator<'_> {
         )
     }
 
+    fn deallocate_funcs(
```

Comment:
A case comes to my mind: Does dealloc functions properly consider the case when opaque field *has not* been filled? For example, suppose a struct/enum has a opaque field. And when the opaque field is being filled, it throws, so it is never filled (and thus have default null value in that field).

For example, this may be solved by, e.g. making Dart opaque understand null pointers.

Reply:
validation delegated to rust wire2api